src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java

```diff
@@ -95,37 +144,49 @@
      * Default is plaintext
      *
      * security.protocol in librdkafka
+     * 
+     * @return The SASL password.
      */
     String password() default "";
 
     /**
      * Gets or sets the security protocol used to communicate with brokers
      * default is PLAINTEXT
+     * 
+     * @return The protocol.
      */
     BrokerProtocol protocol() default BrokerProtocol.NOTSET;
 
     /**
      * Path to client's private key (PEM) used for authentication.
      * Default ""
      * ssl.key.location in librdkafka
+     * 
+     * @return The ssl.key.location.
      */
     String sslKeyLocation() default "";
 
-        /**
+    /**
      * Path to CA certificate file for verifying the broker's certificate.
      * ssl.ca.location in librdkafka
+     * 
+     * @return The ssl ca location.
      */
     String sslCaLocation() default "";
 
     /**
      * Path to client's certificate.
      * ssl.certificate.location in librdkafka
+     * 
+     * @return The client certificate.
```

Comment:
Can we change it to The ssl client certification location?

Reply:
Done